sphinx/themes/basic/static/basic.css_t

```diff
@@ -768,12 +768,16 @@ div.code-block-caption code {
     background-color: transparent;
 }
 
-table.highlighttable td.linenos,
-span.linenos,
```

Comment:
What about the line numbers, why are they excluded?

Reply:
Oh, I misunderstood their roles in the CSS. Going to change it 👍